db/migrate/20250315174325_create_pghero_query_stats.rb

```diff
@@ -0,0 +1,17 @@
+class CreatePgheroQueryStats < ActiveRecord::Migration[8.0]
+  def change
+    # rubocop:disable Rails/CreateTableWithTimestamps
```

Comment:
There's a `captured_at` timestamp, which I guess is enough. I don't want to depart from PgHero's expected data model.